admin_guide/install/first_steps.adoc

```diff
@@ -0,0 +1,77 @@
+
+*Note:* if you are setting up a subscribed system and prefer to use the RHEL7 based images,  run this command instead:
+    
+    $ oc create -f examples/image-streams/image-streams-rhel7.json -n openshift
+
+It is not possible to create both sets of ImageStreams because they use the same name.  If you desite to have both sets of ImageStreams available to users, either create one set in a different Project, or edit one of the files and modify the ImageStream names to make them unique.
+
+== Create ImageStreams for JBoss Middleware Images
+The JBoss Middleware image streams provide images for JBoss EAP, JBoss EWS, and JBoss AMQ.  They can be used to build applications for those platforms or run directly as is.
+
+    $ oc create -f examples/middleware-streams/jboss-image-streams.json
+
+*Note:* Access to the images referenced by these imagestreams requires a subscription. 
+
+== Create Database Service Templates
+The database service templates make it easy to run a database image which can be utilized by other components.  For each database (MongoDB, MySQL, PostgreSQL), two templates are defined.  One uses ephemeral storage in the container which means data stored will be lost if the container is restarted such as because the Pod moves.  This template should be used for demonstration purposes only.  The other template defines a persistent volume for storage, however it requires your OpenShift deployment have persistent volumes configured.
+
+Once registered, users will be able to easily instantiate the various templates, giving them quick access to a database deployment.
+
+To register the core set of database templates, run:
+
+    $ oc create -f examples/db-templates -n openshift
+
+
+== Create QuickStart Templates
+The QuickStart templates define a full set of components for a running application.  These include:
+* BuildConfig to build the application from source located in a GitHub public repository
+* DeploymentConfig to deploy the application image once built
+* Service to provide load balancing for the application Pods
+* Route to provide external access to the application
+
+Some of the templates also define a database deployment and service so the application can perform database operations.
+
+Once registered, users will be able to easily instantiate full applications using the various language images provided with OpenShift.  They can also customize the template parameters during instantiation so that it build source from their own repository rather than the sample repository, so this provides a simple starting point for building new applications.
+
+To register the basic templates run:
+
+    $ oc create -n openshift -f examples/quickstart-templates
+
+There is also a set of templates for creating applications using various JBoss products (EAP, EWS, AMQ) which can be registered by running: 
+
+    $ oc create -n openshift -f examples/middleware-templates
```

Comment:
@sdodson @brenton i took a guess at the path we'll make this available under...  can we either use this path, or we need to update this doc to the actual location.